tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "extends": "@tsconfig/recommended/tsconfig.json",
   "compilerOptions": {
     "outDir": "./dist/",
-    "lib": ["es2015", "dom"],
+    "lib": ["ES2020", "dom"],
```

Comment:
Does this change which browsers we support? Is `ES2020` backwards compatible enough?

Reply:
I added this for Object.entries. Support for which is quite good: https://caniuse.com/object-entries

Whoever uses the library has the option of using webpack to polyfill missing things to improve compat anyway.

Reply:
Ah good point, whoever uses the library will very likely transpile it anyway, so we don't need to be super backwards compatible. 👍